net-vpn/i2p/i2p-2.3.0-r3.ebuild

```
@@ -41,12 +41,13 @@ CP_DEPEND="
 	sys-devel/gettext:0[java]
 	www-servers/tomcat:9
 "
+# jdk-11 for bug #932030
 DEPEND="
 	dev-libs/gmp:0=
 	${CP_DEPEND}
-	>=virtual/jdk-1.8:*
+	>=virtual/jdk-11:*
 	test? (
-		dev-java/ant:0[junit4]
+		>=dev-java/ant-1.10.14-r3:0[junit4]
```

it's the only version available in tree anyway

```suggestion
		dev-java/ant:0[junit4]
```

---

The restriction >=dev-java/ant-1.10.14-r3:0[junit4] is still needed to trigger rebuild of an eventually installed old `ant-1.10.9`.